### DISCUSSION TITLE
Is it possible to use codehike and vitepress together?

### DISCUSSION THREAD

**Discussion by codersjj** [code-hike/codehike#403] [Q&A] [answered]
[Vitepress](https://vitepress.dev/) is a Vite & Vue Powered Static Site Generator.

**Comment by pomber** [comment, marked as answer]
nope, it only works with React

**Reply by codersjj** [reply]
It seems like that, fortunately I found [island.js](https://islandjs.dev/en/), which can be used with Code Hike.:smile:
